_includes/components/404_search.html

```diff
@@ -0,0 +1,9 @@
+<form class="form"  method="GET" action="/search/">
+    <div class="input-group">
+        <input type="text" class="form-control" id="404_search_form" name="q" placeholder="HPC...">
```

Comment:
Can the placeholder be changed to something more helpful?

Reply:
What do you want this to be? "Search linaro.org" ?

Reply:
How about "Enter search term..."?

Reply:
@pcolmer done :+1: